src/main/scala/is/hail/utils/TempDir.scala

```diff
@@ -38,7 +38,7 @@ object TempDir {
       createTempDir("/tmp", hConf))
 }
 
-class TempDir(localTempDir: String, tempDir: String) {
+class TempDir(val localTempDir: String, val tempDir: String) {
```

Comment:
I think these are purposely hidden.

Even though it's called `createTempFile` it just creates a new filename thats not used. You can create a directory with that name if you want a directory instead of a file.

In this case, you can use `hc.tmpDir` since you're in a SparkSuite below.